### ISSUE TITLE
inventory plugin gets "<urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:997)>"

### ISSUE THREAD

**Issue by paulgiordano** [telekom-mms/ansible-collection-icinga-director#219]
### Description

Creating initial test for icinga_director_inventory.yml, yaml file:
```
plugin: telekom_mms.icinga_director.icinga_director_inventory
url: "https://icinga.abt.com.local"
url_username: abticinga
url_password: "xxxxxxxx"
force_basic_auth: False
```
Run with: 
`ansible-playbook -i inventory.icinga_director_inventory.yml t.yml`

t.yml:
```
---
- name: create command template for icinga
  hosts: all
  gather_facts: no

  tasks:

  - name: output the host
    debug:
      msg: "Icinga host {{ inventory_hostname }}"
```

### Reproduction steps

1. Create above files
2. run ansible

### Current Behavior

```
❯ ansible-playbook -i inventory.icinga_director_inventory.yml t.yml
[WARNING]:  * Failed to parse /home/ansible/Ansible/icinga/director/inventory.icinga_director_inventory.yml with auto plugin:
<urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self-signed certificate (_ssl.c:997)>
[WARNING]:  * Failed to parse /home/ansible/Ansible/icinga/director/inventory.icinga_director_inventory.yml with yaml plugin:
Plugin configuration YAML file, not YAML inventory
[WARNING]:  * Failed to parse /home/ansible/Ansible/icinga/director/inventory.icinga_director_inventory.yml with ini plugin:
Invalid host pattern 'plugin:' supplied, ending in ':' is not allowed, this character is reserved to provide a port.
[WARNING]: Unable to parse /home/ansible/Ansible/icinga/director/inventory.icinga_director_inventory.yml as an inventory source
[WARNING]: No inventory was parsed, only implicit localhost is available
[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

PLAY [create command template for icinga] ******************************************************************************************
skipping: no hosts matched

PLAY RECAP *************************************************************************************************************************
```

### Expected Behavior

Hosts fed to ansible play

### Additional information

It appears that the "extra" parameters like verify_certs, client_cert, and client_key are not implemented, though in the docs

**Comment by rndmh3ro** [comment]
You're right. That's a bug. We'll have to fix this.

**Comment by rndmh3ro** [comment]
Hey @paulgiordano,

I started working on it here: https://github.com/telekom-mms/ansible-collection-icinga-director/pull/221

Since I cannot test this right, is it possible for you to test it?

**Comment by rndmh3ro** [comment]
I got the tests working. I'll have to update them though.

**Comment by paulgiordano** [comment]
I went ahead and tested in my use-case, and it works after I deleted a duplicate line (line 99) and commented out the "force" line. Thanks for looking at this so quickly!

**Comment by rndmh3ro** [comment]
Fixed in #221 and released in in [1.34.1](https://github.com/telekom-mms/ansible-collection-icinga-director/releases/tag/1.34.1)
